### PR TITLE
Feature/UI

### PR DESCRIPTION
**Pull request by byiringiroscar**
####  Add UI
This PR is raised to implement the following features 🆗 :

- [x] Add UI
- [x] Setup linters
- [x] Use modules

### How should it be tested ⬇️ ⬇️ ⬇️ 

clone this repository
     
```
git clone https://github.com/byiringiroscar/OOP-school-library-with-Ruby/
```
Navigate to folder
```
cd OOP-school-library-with-Ruby
```
Switch to this feature/ui
```
git switch feature/ui
```
Run project and Installation
- open the command prompt and run :
```
ruby main.rb
```

### Thanks for reviewing my PR 😄

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    end
+  end
+
+  def create_rental
+    puts 'Select a person from the following list by number (not ID):'
+
+    @allpeople.each_with_index do |person, index|
+      person_type = person.is_a?(Student) ? '[Student]' : '[Teacher]'
+      puts "#{index}) #{person_type} Name: #{person.name.chomp}, ID: #{person.id}, Age: #{person.age}"
+    end
```
Comment (CollinsTatang) [inline app.rb]: - [ ] Please fix this error your app generates when a user tries to create a rental
Screenshot 2023-05-27 at 22 16 10

File: person.rb [hunk 2]
```diff
@@ -31,3 +31,6 @@ def of_age?
     @age.to_i >= 18
   end
 end
+
+# person = Person.new(22, 'maximilianus', parent_permission: false)
+# puts person.parent_permission
```
Comment (CollinsTatang) [inline person.rb]: - [ ] Please remove all commented code from your codebase to make your code clean and easy to maintain.

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @byiringiroscar, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🥳

Hi @byiringiroscar 🙋‍♀️

Great work implementing the changes suggested by the previous reviewer 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Great UI, well done!
-  ✅ Great work using the right Gitflow 
-  ✅ Linters are working properly, great job! 

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**